docker/Dockerfile-pgbase

```diff
@@ -0,0 +1,15 @@
+FROM python:2.7
+RUN apt-get update && apt-get install -y \
+    libffi-dev \
+    libssl-dev \
+    postgresql-client
```

Comment:
I'd add `rm -rf /var/lib/apt/lists/*` here to clean up after the apt-get update - like the official images do:
https://github.com/docker-library/python/blob/a4ce69de053337ec22d7f322cd997398eeba2350/2.7/stretch/slim/Dockerfile#L17